qiskit/providers/__init__.py

```diff
@@ -240,21 +263,11 @@ def run(circuits, **kwargs):
 --------------------
 
 The key piece of the backend object is how it describes itself to the compiler.
-This is composed of three classes, the
-:class:`~qiskit.providers.models.BackendConfiguration` which describes the
-immutable properties of the backend (things like the number of qubits, coupling
-map, etc), the :class:`~qiskit.providers.models.BackendProperties` which
-describes measured properties of the device (such as current error rates), and
-the :class:`~qiskit.providers.models.PulseDefaults` which defines the
-default pulse behavior for the backend (such as the pulse sequences for gates).
-Of these only :class:`~qiskit.providers.models.BackendConfiguration` is
-required.
-
-It's worth noting that this interface will likely evolve over time as the
-compiler grows and improves, thus requiring new information from the backends.
-This is a key reason why the providers interfaces are versioned to enable
-making these changes in a controlled manner. The details here will change and
-reflect the latest version of this interface.
+This is handled with the :class:`~qiskit.transpiler.Target` class which defines
+a model of a backend for the transpiler. A backend object will need to return
+a :class:`~qiskit.transpiler.Target` object from the :attr:`~qiskit.providers.BackendV2.target`
+attribute which the :func:`~qiskit.compiler.transpile` function will use as
+it's model of a backend target for compilation.
```

Comment:
its model

Reply:
Fixed in the current revision (just forced pushed as part of a rebase too)